### DISCUSSION TITLE
Trade Items List

### DISCUSSION THREAD

**Discussion by Kamiikaze** [xanthics/bdo_ship_upgrade#9] [Ideas]
As multiple ship material items can be traded with multiple trade items, a combined list of all trade items would be nice.

So its easier to track which trade items to keep and which can be freely used to exchange it for higher one.

**Comment by xanthics** [comment]
To clarify, you mean a page that lists all possible trade items in the first column and all the things you can trade them for in the second column?  And for all trade items or only ones this guide considers?

**Reply by Kamiikaze** [reply]
I mean on the bottom of the page, there is a list of missing ship materials and which trade goods you can exchange to obtain that specific ship material.

So when doing bartering to fill up the stock, it would be easier to have a list which trade goods to go for.

**Reply by xanthics** [reply]
So just an extra section at the end that lists each trade good once that is used at least once for an item missing from the guide?

**Reply by Kamiikaze** [reply]
Yes, exactly 😊

**Reply by xanthics** [reply]
Something like this?

![image](https://github.com/user-attachments/assets/ec9ddac3-41a6-4692-92f1-e021dd3f44a8)

**Comment by xanthics** [comment]
https://github.com/xanthics/bdo_ship_upgrade/commit/c4c76c28013762f77e21d2a38097fc679de2be5e
